sdk/logs/src/main/java/io/opentelemetry/sdk/logs/data/Body.java

```diff
@@ -31,8 +32,18 @@ static Body stringBody(String stringValue) {
     return StringBody.create(stringValue);
   }
 
+  /**
+   * Return an empty {@code Body}.
+   *
+   * @return a {@code Body} without a value.
+   */
+  static Body emptyBody() {
```

Comment:
```suggestion
  static Body empty() {
```

Reply:
Some APIs may repeat the type name in factories for static import but I guess we haven't been

Reply:
I like this better. I think we can change `stringBody(String)` to just `string(String)` to match this.